src/app/tests/suites/certification/PICS.yaml

```diff
@@ -6289,6 +6289,119 @@ PICS:
     - label: "Read all supported mandatory attribute"
       id: S.C.AM-READ
 
+    # Smoke CO Alarm Cluster Test Plan
+    - label: "Does the device implement the SMCO cluster as a server?"
+      id: SMOKECO.S
+
+    - label: "Does the device implement the SMCO cluster as a client?"
+      id: SMOKECO.C
+
+    #
+    # server / features
+    #
+    - label: "Does the device support this specific cluster feature?"
+      id: SMOKECO.S.F00
+
+    - label: "Does the device support this specific cluster feature?"
+      id: SMOKECO.S.F01
+
+    #
+    # server / attributes
+    #
+    - label: "Does the device implement the ExpressedState attribute?"
+      id: SMOKECO.S.A0000
+
+    - label: "Does the device implement the SmokeState attribute?"
+      id: SMOKECO.S.A0001
+
+    - label: "Does the device implement the COState attribute?"
+      id: SMOKECO.S.A0002
+
+    - label: "Does the device implement the BatteryAlert attribute?"
+      id: SMOKECO.S.A0003
+
+    - label: "Does the device implement the DeviceMuted attribute?"
+      id: SMOKECO.S.A0004
+
+    - label: "Does the device implement the TestInProgress attribute?"
+      id: SMOKECO.S.A0005
+
+    - label: "Does the device implement the HardwareFaultAlert attribute?"
+      id: SMOKECO.S.A0006
+
+    - label: "Does the device implement the EndOfServiceAlert attribute?"
+      id: SMOKECO.S.A0007
+
+    - label: "Does the device implement the InterconnectSmokeAlarm attribute?"
+      id: SMOKECO.S.A0008
+
+    - label: "Does the device implement the InterconnectCOAlarm attribute?"
+      id: SMOKECO.S.A0009
+
+    - label: "Does the device implement the ContaminationState attribute?"
+      id: SMOKECO.S.A000a
+
+    - label: "Does the device implement the SensitivityLevel attribute?"
```

Comment:
It's SmokeSensitivityLevel in the spec, no?  Test plan issue needed.  Also, the test plan has the wrong conformance listed for this attribute.